docs/building-basics.rst

```diff
@@ -0,0 +1,41 @@
+Building Concepts and Setup
+===========================
+
+The :doc:`introduction` and page on :doc:`first-build` have already shown how applications get built as Flatpaks, and have introduced many of the core ideas. This page describes some of these ideas and concepts in more detail. In doing so, it provides guidance on picking a runtime, getting setup to build applications, and when to bundle dependencies yourself.
+
+Runtimes
+--------
+
+As was described in the :doc:`introduction`, runtimes provide basic dependencies that can be used by applications. They also provide the environment that applications run in.
+
+Flatpak requires each application to specify a runtime, and this runtime must be present on a system for it to run. Therefore, one of the first decisions you need to make when building an application with Flatpak, is which runtime it will use.
+
+An overview of the runtimes that are available can be found in the :doc:`available-runtimes` page. There are deliberately only a small number of runtimes to choose from. Typically, runtimes are picked on the basis of which dependencies an application requires. If a runtime exists that provides libraries that you plan on using, this is usually the correct runtime to use!
+
+.. tip::
+
+  Runtimes require regular maintenance, and application developers should generally not consider creating their own.
+
+Once you have chosen a runtime for your application, you will need to install it. Runtimes are installed in exactly the same way as applications, with the ``flatpak install`` command. For example, the command to install the GNOME 3.26 runtime is::
```

Comment:
It may be worth noting end users do not need to manually install runtimes. And actually developers don't need to either if they specify `--install-deps-from=flathub` when using `flatpak-builder`.

Reply:
Fixed.